packages/themes/src/types/index.ts

```diff
@@ -6,10 +6,10 @@ export type Dictionary<T = any> = Record<string, T>;
 
 type ThemeStyleResolve<T extends ComponentNames, K> = ComponentType<T> extends K
   ? keyof ComponentType<T>[keyof K]
-  : never;
+  : string;
```

Comment:
ini udah bagus pake never sih, misal ada komponen yang belum define variants, si ts bakal mark ini sebagai error "ehh komponen lo belom ada variant nih, kenapa di isi string". kalo begini jadi kurang powerful sih (kurang safe).

![Screenshot 2022-11-29 at 09 55 42](https://user-images.githubusercontent.com/54368620/204427872-23d553df-476b-487b-8245-56d13f787a49.png)

Reply:
fixed in #38 [ef2310f](https://github.com/yoru-ui/yoru-ui/pull/38/commits/ef2310f9453e547353231d10b09e850fb29d50a4)